apps/client/src/app/[locale]/(protected)/dashboard/components/PendingOrders.tsx

```diff
@@ -0,0 +1,34 @@
+"use client";
+
+import React from 'react';
+import { MdTrendingUp } from 'react-icons/md';
+
+const PendingOrders = () => (
```

Comment:
okay, so all 4 cards/charts looks fairly similar in style and functionality yeah? maybe try to make them a component with different variants? 

then on top of that you could also consider making them "stateless" by taking in data props. An idea would look something like this

```tsx
type Props = {
  title: string,
  subtitle?: string,
  theme: "red" | "blue" | "green" // or how ever want to call this?
} & (
  {
    type: "percentage"
    percentage: number
  } | {
    type: "bar-chart"
    someData: ????
  }
)
```


let me know if you need more clarification on this